connect/runtime/src/test/java/org/apache/kafka/connect/converters/BooleanConverterTest.java

```diff
@@ -74,10 +74,12 @@ public void testFromConnectNull() {
     @Test
     public void testToConnect() {
         assertEquals(Schema.OPTIONAL_BOOLEAN_SCHEMA, converter.toConnectData(TOPIC, TRUE).schema());
-        assertEquals(Schema.OPTIONAL_BOOLEAN_SCHEMA, converter.toConnectData(TOPIC, FALSE).schema());
-        assertEquals(Schema.OPTIONAL_BOOLEAN_SCHEMA, converter.toConnectData(TOPIC, null).schema());
         assertTrue((Boolean) converter.toConnectData(TOPIC, TRUE).value());
+
+        assertEquals(Schema.OPTIONAL_BOOLEAN_SCHEMA, converter.toConnectData(TOPIC, FALSE).schema());
```

Comment:
The original test validates the value, is there a reason to remove it?

Reply:
I'm sorry I don't quite follow your question. Which test are you referring to?

Reply:
Oh sorry if I didn't make it clear. Looks like the previous commit the value is also asserted via the `AssertEquals`, as well as the converter's output via `assertTrue/false`, but then the value comparison was removed. Seems now it's added back again. 👍